Gems/MiniAudio/Code/Source/Clients/MiniAudioPlaybackComponentConfig.cpp

```diff
@@ -24,9 +24,27 @@ namespace MiniAudio
                 ->Field("Auto-follow", &MiniAudioPlaybackComponentConfig::m_autoFollowEntity)
                 ->Field("Loop", &MiniAudioPlaybackComponentConfig::m_loop)
                 ->Field("Spatialization", &MiniAudioPlaybackComponentConfig::m_enableSpatialization)
+                ->Field("Fixed Direction", &MiniAudioPlaybackComponentConfig::m_fixedDirection)
+                ->Field("Direction", &MiniAudioPlaybackComponentConfig::m_direction)
                 ->Field("Attenuation Model", &MiniAudioPlaybackComponentConfig::m_attenuationModel)
+                ->Field("Directional Attenuation Factor", &MiniAudioPlaybackComponentConfig::m_directionalAttenuationFactor)
                 ->Field("Min Distance", &MiniAudioPlaybackComponentConfig::m_minimumDistance)
                 ->Field("Max Distance", &MiniAudioPlaybackComponentConfig::m_maximumDistance)
+                ->Field("Inner Cone Angle (Degrees)", &MiniAudioPlaybackComponentConfig::m_innerAngleInDegrees)
+                    ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
+                    ->Attribute(AZ::Edit::Attributes::Step, 1.0f)
+                    ->Attribute(AZ::Edit::Attributes::Max, 360.0f)
+                    ->Attribute(AZ::Edit::Attributes::Suffix, " degrees")
+                ->Field("Outer Cone Angle (Degrees)", &MiniAudioPlaybackComponentConfig::m_outerAngleInDegrees)
+                    ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
+                    ->Attribute(AZ::Edit::Attributes::Step, 1.0f)
+                    ->Attribute(AZ::Edit::Attributes::Max, 360.0f)
+                    ->Attribute(AZ::Edit::Attributes::Suffix, " degrees")
+                ->Field("Outer Volume (%)", &MiniAudioPlaybackComponentConfig::m_outerVolume)
```

Comment:
nit: any reason why the label also shows (Degrees) or (%) when it's also displayed in the suffix?

Reply:
@AMZN-alexpete I got rid of the redundancy in the label and kept the suffix, see fb4f8af6cb144834cc12b9e154e1340250d7cbff. Thanks for the feedback.